src/xfastertransformer/tools/convert.py

```diff
@@ -53,16 +53,22 @@ def get_weight_data_type(self, dtype: str):
         else:
             raise Exception(f"{self.__class__.__name__} don't support convert weight to {dtype}.")
 
-    def convert(self, input_dir, output_dir=None, dtype: str = "fp16", processes=8):
+    def convert(self, input_dir, output_dir=None, dtype: str = "fp16", processes=8, quantization=None):
```

Comment:
\# from_quantized_model is from HF int8 model to xFT int8 model.
def convert(self, input_dir, output_dir=None, dtype: str = "fp16", processes=8, from_quantized_model=None):